scripts/test-integrations.sh

```diff
@@ -0,0 +1,164 @@
+    if [ "`cat HEAD.sha`" != "$SHA" ]; then
+      # Rebuild if HEAD has changed since last time
+      cp server/config/local.json-dist server/config/local.json
+      npm i
+      CONFIG_FILES=server/config/local.json,server/config/production.json node_modules/.bin/grunt build
+    fi
+    # Save the SHA of HEAD for next time
+    echo "$SHA" > HEAD.sha
+
+    if [ -e fxa-oauth-server ]; then
+      # Update
+      cd fxa-oauth-server
+      git fetch --depth 1
+      git reset --hard origin/master
+    else
+      # Create
+      git clone --depth 1 https://github.com/mozilla/fxa-oauth-server
+      cd fxa-oauth-server
+    fi
+
+    # Get the SHA of HEAD
+    SHA=`git log | head -1 | cut -d ' ' -f2`
+    if [ "`cat HEAD.sha`" != "$SHA" ]; then
+      # Rebuild if HEAD has changed since last time
+      npm i --production
+    fi
+    # Save the SHA of HEAD for next time
+    echo "$SHA" > HEAD.sha
+
+    if [ -e fxa-profile-server ]; then
+      # Update
+      cd fxa-profile-server
+      git fetch --depth 1
+      git reset --hard origin/master
+    else
+      # Create
+      git clone --depth 1 https://github.com/mozilla/fxa-profile-server
+      cd fxa-profile-server
+    fi
+
+    # Get the SHA of HEAD
+    SHA=`git log | head -1 | cut -d ' ' -f2`
+    if [ "`cat HEAD.sha`" != "$SHA" ]; then
+      # Rebuild if HEAD has changed since last time
+      npm i --production
+    fi
+    # Save the SHA of HEAD for next time
+    echo "$SHA" > HEAD.sha
+
+    if [ -e browserid-verifier ]; then
+      # Update
+      cd browserid-verifier
+      git fetch --depth 1
+      git reset --hard origin/http
+    else
+      # Create
+      git clone --depth 1 --branch http https://github.com/vladikoff/browserid-verifier
+      cd browserid-verifier
+    fi
+
+    # Get the SHA of HEAD
+    SHA=`git log | head -1 | cut -d ' ' -f2`
+    if [ "`cat HEAD.sha`" != "$SHA" ]; then
+      # Rebuild if HEAD has changed since last time
+      npm i --production
+      npm i vladikoff/browserid-local-verify#http
+    fi
+    # Save the SHA of HEAD for next time
+    echo "$SHA" > HEAD.sha
+
+    if [ ! -e firefox ]; then
+      pip install mozdownload mozinstall
+      mozdownload --version 50.1.0 --destination firefox.tar.bz2
+      mozinstall firefox.tar.bz2
+    fi
+
+    ulimit -S -n 2048
+
+    mkdir -p $HOME/.vnc
+    bash fxa-content-server/tests/ci/setvncpass.sh
+    tightvncserver :1
+    export DISPLAY=:1
+
+    npm i -g retry-cli
+    ;;
+
+  deps)
+    npm i
+    ;;
+
+  post-deps)
+    SIGNIN_UNBLOCK_ALLOWED_EMAILS="^block.*@restmail\\.net$" SIGNIN_UNBLOCK_FORCED_EMAILS="^block.*@restmail\\.net$" npm start 2>&1 | tee $HOME/fxa-auth-server.log &
+
+    cd fxa-content-server && CONFIG_FILES=server/config/local.json,server/config/production.json node_modules/.bin/grunt serverproc:dist 2>&1 | tee $HOME/fxa-content-server.log &
+
+    cd fxa-oauth-server && LOG_LEVEL=error NODE_ENV=dev node ./bin/server 2>&1 | tee $HOME/fxa-oauth-server.log &
```

Comment:
I wasn't certain if the ` &` suffix was enough to mimic the `background` directive for Circle...